tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -172,6 +172,32 @@ def test_positive_create_with_description_by_cv_id(self):
         })
         self.assertEqual(cvf['description'], description)
 
+    @tier1
+    def test_positive_list_by_name_and_org(self):
+        """Create new content view filter and try to list it by its name and
+        organization it belongs
+
+        :id: e685892d-9dc3-48f2-8a09-8f861dceaf4e
+
+        :expectedresults: Content view filter created and listed successfully
+
+        :BZ: 1378018
+
+        :CaseImportance: Critical
+        """
+        cvf_name = gen_string('utf8')
+        ContentView.filter.create({
+            'content-view-id': self.content_view['id'],
+            'name': cvf_name,
+            'organization-id': self.org['id'],
+            'type': 'package_group',
+        })
+        cvf = ContentView.filter.list({
+            u'content-view': self.content_view['name'],
+            u'organization': self.org['name'],
+        })
+        self.assertEqual(cvf[0]['name'], cvf_name)
```

Comment:
additional step needed before this line
self.assertEqual(len(cvf), 1)
or 
self.assertGreater(len(cvf), 0)